### PR TITLE
chore: upgrade of deps

### PR DESCRIPTION
**Pull request by bogdan-bashev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,16 +14,16 @@
...
-    "lerna": "^4.0.0",
-    "semantic-release": "^18.0.1",
-    "semantic-release-monorepo": "^7.0.4",
-    "typescript": "^4.0.3"
+    "eslint": "^8.36.0",
+    "husky": "^8.0.3",
+    "lerna": "^6.5.1",
+    "semantic-release": "^20.1.3",
+    "semantic-release-monorepo": "^7.0.5",
+    "typescript": "^5.0.2"
```
Comment (madisp) [inline package.json:26]: will this enforce ts 5.x on downstream deps? :P I don't mind this, but if we do I think the version should be at least `0.4.0`

Comment (bogdan-bashev) [reply package.json:26]: bumped to 0.4.0

**Review by madisp — APPROVED** [review]

**Comment by madisp** [comment]
fixed the CircleCI pipeline, config error from our side

**Review by eritikass — APPROVED** [review]

**Pull request merged**